simde/arm/neon/cmla.h

```diff
@@ -0,0 +1,371 @@
+   #if defined(SIMDE_ARM_NEON_A32V8_NATIVE)
+    return vcmlaq_f32(r, a, b);
+   #else
+    simde_float32x4_private
+      r_ = simde_float32x4_to_private(r),
+      a_ = simde_float32x4_to_private(a),
+      b_ = simde_float32x4_to_private(b);
+
+    SIMDE_VECTORIZE
+    for(size_t i = 0 ; i < (sizeof(r_.values) / sizeof(r_.values[0])) ; i++) {
+      if(i % 2 == 0)
+        r_.values[i] += b_.values[i] * a_.values[i];
+      else
+        r_.values[i] += b_.values[i] * a_.values[i-1];
+    }
+
+    return simde_float32x4_from_private(r_);
+   #endif
+ }
+ #if defined(SIMDE_ARM_NEON_A32V8_ENABLE_NATIVE_ALIASES)
+   #undef vcmlaq_f32
+   #define vcmlaq_f32(r, a, b) simde_vcmlaq_f32(r, a, b)
+ #endif
+
+ SIMDE_FUNCTION_ATTRIBUTES
+ simde_float64x2_t
+ simde_vcmlaq_f64(simde_float64x2_t r, simde_float64x2_t a, simde_float64x2_t b) {
+   #if defined(SIMDE_ARM_NEON_A32V8_NATIVE)
+    return vcmlaq_f64(r, a, b);
+   #else
+    simde_float64x2_private
+      r_ = simde_float64x2_to_private(r),
+      a_ = simde_float64x2_to_private(a),
+      b_ = simde_float64x2_to_private(b);
+
+    SIMDE_VECTORIZE
+    for(size_t i = 0 ; i < (sizeof(r_.values) / sizeof(r_.values[0])) ; i++) {
+      if(i % 2 == 0)
+        r_.values[i] += b_.values[i] * a_.values[i];
+      else
+        r_.values[i] += b_.values[i] * a_.values[i-1];
+    }
+
+    return simde_float64x2_from_private(r_);
+   #endif
+ }
+ #if defined(SIMDE_ARM_NEON_A32V8_ENABLE_NATIVE_ALIASES)
+   #undef vcmlaq_f64
+   #define vcmlaq_f64(r, a, b) simde_vcmlaq_f64(r, a, b)
+ #endif
+
+ /* * rot 90 * */
+
+ SIMDE_FUNCTION_ATTRIBUTES
+ simde_float32x2_t
+ simde_vcmla_rot90_f32(simde_float32x2_t r, simde_float32x2_t a, simde_float32x2_t b) {
+   #if defined(SIMDE_ARM_NEON_A32V8_NATIVE)
+    return vcmla_rot90_f32(r, a, b);
+   #else
+    simde_float32x2_private
+      r_ = simde_float32x2_to_private(r),
+      a_ = simde_float32x2_to_private(a),
+      b_ = simde_float32x2_to_private(b);
+
+    SIMDE_VECTORIZE
+    for(size_t i = 0 ; i < (sizeof(r_.values) / sizeof(r_.values[0])) ; i++) {
+      if(i % 2 == 0)
+        r_.values[i] += -(b_.values[i+1]) * a_.values[i+1];
+      else
+        r_.values[i] += b_.values[i-1] * a_.values[i];
+    }
+
+    return simde_float32x2_from_private(r_);
+   #endif
+ }
+ #if defined(SIMDE_ARM_NEON_A32V8_ENABLE_NATIVE_ALIASES)
+   #undef vcmla_rot90_f32
+   #define vcmla_rot90_f32(r, a, b) simde_vcmla_rot90_f32(r, a, b)
+ #endif
+
+ SIMDE_FUNCTION_ATTRIBUTES
+ simde_float32x4_t
+ simde_vcmlaq_rot90_f32(simde_float32x4_t r, simde_float32x4_t a, simde_float32x4_t b) {
+   #if defined(SIMDE_ARM_NEON_A32V8_NATIVE)
+    return vcmlaq_rot90_f32(r, a, b);
+   #else
+    simde_float32x4_private
+      r_ = simde_float32x4_to_private(r),
+      a_ = simde_float32x4_to_private(a),
+      b_ = simde_float32x4_to_private(b);
+
+    SIMDE_VECTORIZE
+    for(size_t i = 0 ; i < (sizeof(r_.values) / sizeof(r_.values[0])) ; i++) {
```

Comment:
Another small style nitpick (sorry): there should be a space between `for` and the open-parenthesis.  Same thing with the if statements.